services/task-svc/internal/repositories/patient_repository.go

```diff
@@ -83,7 +83,8 @@ func (r *PatientRepository) GetUnassignedPatientsForOrganization(organizationID
 func (r *PatientRepository) GetDischargedPatientsForOrganization(organizationID uuid.UUID) ([]models.Patient, error) {
 	var patients []models.Patient
 	query := r.db.
-		Where("organization_id = ? AND is_discharged = 1", organizationID).
+		Unscoped().
+		Where("organization_id = ? AND NOT is_discharged = 0", organizationID).
```

Comment:
Why did you negate this condition, and why is this change in the scope of this pull request?

Reply:
because the request didn't return any discharged patients, because we need the unscoped to see the soft deleted entries.
The `NOT is_discharged = 0` is there, because the softdelete is a timestamp and any value that isn't the default (not deleted) indicates deletion. Thus checking `discharged = 0` might not work properly